gtk/src/toga_gtk/widgets/tree.py

```diff
@@ -15,6 +15,7 @@ def create(self):
         self.store = SourceTreeModel(
             [{"type": str, "attr": a} for a in self.interface._accessors],
             is_tree=is_tree,
+            missing_value=getattr(self.interface, "missing_value", None),
```

Comment:
Why use getattr here? Unless I'm missing something, `missing_value` should always be defined on the interface.

Reply:
I am not at my computer to confirm fully but I believe the issue was the Tree interface did not have the missing_value. If you are able to look at the first round of checks it should show which interface did have it

Reply:
Ah - that makes sense. That's an API inconsistency that should probably be resolved...